tests/fx/converters/test_reduce_ops_fx.py

```diff
@@ -100,3 +100,35 @@ def test_cumsum(dim):
     for mod in [mod_func, mod_method]:
         mgx_mod = convert_to_mgx(mod, [inp])
         verify_outputs(mod, mgx_mod, inp)
+
+
+@pytest.mark.parametrize('dim, keepdim', [(0, True), (-1, False),
+                                          ([2, 3], False), (None, None)])
```

Comment:
```suggestion

@pytest.mark.skip_min_migraphx_ver("2.11.0")
@pytest.mark.parametrize('dim, keepdim', [(0, True), (-1, False),
                                          ([2, 3], False), (None, None)])
```

With #161 you can add this to skip these tests